### ISSUE TITLE
Zone Telechargement

### ISSUE THREAD

**Issue by ludopotte** [Kodi-vStream/venom-xbmc-addons#3342]
Depuis un moment le Site Zone Téléchargement marche mal et vous pouvez contrôler son fonctionnement au https://downforeveryoneorjustme.com/zt-za.net et pour accéder au site c’est via www.zt-za.net (accessible via VPN, 
en Espagne par exemple) ou www.zone-telechargement.blue que les utilisateurs doivent se connecter automatiquement.

**Comment by alKODIque** [comment]
Bonjour,
Je crois que ZT a une nouvelle adresse (.cam) c'est affiché sur leur page d'accueil

**Comment by ludopotte** [comment]
Ok moi j'ai pas accès au site et pourrais tu me donner l'adresse complète s'il te plait ?

**Comment by alKODIque** [comment]
Voilà: 
![Capture](https://user-images.githubusercontent.com/49578130/127782071-54b71788-11e0-4e0e-9af4-fa1d0cc8edc4.PNG)

**Comment by ludopotte** [comment]
Merci mais chez moi, y est toujours pas accès.

**Comment by Niconimes** [comment]
salut, il faut que tu changes ton DNS pour y avoir accès

**Comment by ludopotte** [comment]
Ok merci

**Comment by alKODIque** [comment]
Effectivement.. et pour vérifier si le changement DNS  a bel et bien été effectué, tu peux vérifier ta config ici: http://www.whatsmydnsserver.com/

**Comment by ludopotte** [comment]
Merci pour toutes ces informations messieurs, ca m'aide beaucoup.

**Comment by sizanic** [comment]
> Merci pour toutes ces informations messieurs, ca m'aide beaucoup.

(Il me semble que alKODIque est une madame 😉 )
